src/templates/en/2019/table_of_contents.html

```diff
@@ -0,0 +1,116 @@
+{% extends "en/2019/base.html" %}
+
+{% block title %}
+Table of Contents | Web Almanac 2019
+{% endblock %}
+
+{% block styles %}
+{{ super() }}
+<style>
+.main {
+  margin-bottom: 100px;
+}
+.title::before {
+  display: block;
+  content: '';
+  margin-bottom: 18px;
+  border-bottom: solid 1px #c6dcd2;
+  width: 70px;
+}
+.contents-wrapper {
+  display: grid;
+  grid-template-areas: 'parts characters';
+}
+.parts {
+  grid-area: parts;
+  max-width: 600px;
+}
+.characters {
+  grid-area: characters;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+.character {
+  max-width: 150px;
+  height: auto;
+}
+.part,
+.part-name,
+.chapter {
+  border-bottom: 1px solid #eee;
+  margin: 0;
+}
+.chapter:last-child {
+  border-bottom: none;
+}
+
+.part-name,
+.chapter {
+  padding: 20px 0;
+}
+
+.chapters {
+  padding-left: 20px;
+}
+
+.chapter .todo {
+  color: #aaa;
+}
+.chapter a {
+  text-decoration: none;
+  color: #1A2B49;
+}
+.chapter a:hover {
```

Comment:
@rviscomi 
there reason why there is not `:focus`?
I like to align behavior to methodology page.

And, is `outline: none;` unnecessary for list? 
(maybe also involves the methodology index)

Reply:
Good catch. Updated.